.github/workflows/ci.yml

```diff
@@ -9,6 +9,7 @@ on:
 permissions:
   actions: read
   contents: read
+  checks: write
```

Comment:
can we add setup-rust to the license check step?

Reply:
Are the latest changes what you had in mind?